### DISCUSSION TITLE
essay/2024/08/08/defense-fun

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#230] [Comments]
# essay/2024/08/08/defense-fun

누구나 축구할 때 공격을 하고 싶어 합니다. 제일 재밌으니까. 모든 스포트라이트를 받으니까. 골을 넣는 것만큼 짜릿한 게 없으니까.

https://jeho.page/essay/2024/08/08/defense-fun.html

**Comment by yangtaeho** [comment]
저희 개발 사부님이 그러셨음. 개발자 유입되는 뉴비들은 공격수 개발자만 개발자인 줄 알아서, 시장에 널리고 널린 게 공격수 개발자라고 ㅎㅎ 물론 그 중에도 뛰어난 분들이 당연히 많지만, 정말 귀하고 필요한 건 수비수 개발자라고요.

**Reply by BenjaminKim** [reply]
좋은 수비수 개발자 하나 있으면 서비스 운영할 때 정말 든든할 것 같습니다.

**Comment by netnabi** [comment]
저도 수비수를 담당하고 있습니다
수비만 10년이 넘는거 같네요
신입때는 공격도 했던거 같은데 서비스 굴러가고 규모가 커지니까 다음 신입, 중급 분들을 키우려니까 수비도 이제 꽤 힘들군요
그래서 가끔 라떼는 말이지와 함께
여러분이 공격할 수 있는것은 수비수가 있기 때문이다
당신들을 받쳐주는 수비수들을 항상 생각하고
그들이 있기에 당신이 있다라고 잔소리를 하고 있습니다
다들 화이팅입니아

**Reply by BenjaminKim** [reply]
이야기를 듣고 보니 경력이 쌓여갈수록 다들 비슷한 마음인 것 같습니다.
모든 수비수분들 화이팅입니다!

**Comment by HanJaeJoon** [comment]
현재는 1인 개발자시니깐 공격, 수비, 감독까지 다 하시고 계시겠네요 ㅎㅎ

**Reply by BenjaminKim** [reply]
하하하, 정말 그렇네요. 😁
